### PR TITLE
update-lesson-2

### PR DESCRIPTION
**Pull request by alexduf**

### CODE REVIEW

File: lessons/lesson-2.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ You should get a list of Camembert, Roquefort, and Cheddah.
 You probably used a simple select statement with a where clause
 checking price.
 
-But what if the price of Cheddah changes frequently and you don't know
+But what if the price of Camembert changes frequently and you don't know
```
Comment (alexduf) [inline lessons/lesson-2.md:60]: this is to be consistent with L50

File: lessons/lesson-2.md [hunk 2]
```diff
@@ -78,24 +78,24 @@ Add the following table:
...
 ```
 
 And add some data:
 
 ```sql
-INSERT INTO orders (product_no, date) VALUES
-    (1, now()),
-    (2, now()),
-    (2, now() - interval '1 hour');
+INSERT INTO orders (product_no, date, quantity) VALUES
```
Comment (alexduf) [inline lessons/lesson-2.md:89]: Added a column for the second to last exercise

File: lessons/lesson-2.md [hunk 3]
```diff
@@ -120,6 +120,12 @@ The docs on these are
 
 >**Exercise:** find the min and max price *in the same query*
 
+## Putting it together
+
+Now you should know how to apply a filter, aggregate and join.
+
+>**Exercise:** find the list of product, their name, and what quantity has been ordered so far
```
Comment (nicl) [inline lessons/lesson-2.md]: product**s** (and names, quantities have)

**Review by nicl — APPROVED** [review]
🎉

**Pull request merged**